docs/pages/eas-update/faq.mdx

```diff
@@ -16,6 +12,19 @@ App Store rules regularly change and just like how you might need to follow them
 
 EAS Update is a great way to quickly deliver improvements to the people who use your apps. For example, consider an app that has a critical bug that needs to be fixed. With EAS Update you can quickly get the fix out and later follow up with a new submission that includes the fix built in.
 
+### How are "monthly updated users" counted for a billing cycle
+
+> **info** 1 monthly updated user = 1 unique installation of an app that downloads at least 1 update during your billing cycle
```

Comment:
```suggestion
> **info** 1 monthly updated user equals 1 unique installation of an app that downloads at least 1 update during your billing cycle.
```

- Let's avoid using symbols as per our [writing style guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#symbols-as-words)
- Missing period at the end of the sentence.